### PR TITLE
✨  Display cast in movie details page

### LINKED ISSUE #67: Display cast in movie details page
Map cast from API to display the cast.

image

### PR DESCRIPTION
**Pull request by l-goede**
Fixes #67

-Build Cast component to map cast
-Add component to movie details page
-add cast to storybook

### CODE REVIEW

**Review by Maurizio-F — COMMENTED** [review]
lgtm 💯

**Review by J-Bannach — APPROVED** [review]
lgtm 💯

File: src/app/components/Cast/Cast.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    profilePath: string;
+    name: string;
+  }>;
+};
+
+function Cast({ actors }: CastProps): JSX.Element {
+  return (
+    <div className={styles.artistCards}>
+      {actors.map((actor) => (
+        <ArtistCard imgSrc={actor.profilePath} artistName={actor.name} />
```
Comment (lajanzen) [inline src/app/components/Cast/Cast.tsx]: We forgot the key :)

**Review by lajanzen — CHANGES_REQUESTED** [review]

**Review by daebert — APPROVED** [review]
good!

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

**Review by fgottesmann — APPROVED** [review]

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Review by infoucarsinan — APPROVED** [review]

**Review by mpagels — APPROVED** [review]

**Review by roabram — APPROVED** [review]
lgtm :)

**Review by lajanzen — APPROVED** [review]

**Review by daebert — APPROVED** [review]
lgtm!

**Review by jujuzitroni — APPROVED** [review]
🚀

**Review by tmh9x — APPROVED** [review]

**Review by infoucarsinan — APPROVED** [review]

**Review by JGieseWeb — APPROVED** [review]
lgtm 👌

**Review by SGierszewski — APPROVED** [review]
Good job! 🥳

**Review by MaxMuecket — APPROVED** [review]
lgtm 💥

**Review by daebert — APPROVED** [review]

**Pull request merged**